autogen/agentchat/contrib/gpt_assistant_agent.py

```diff
@@ -53,9 +53,14 @@ def __init__(
                 - Other kwargs: Except verbose, others are passed directly to ConversableAgent.
         """
         # Use AutoGen OpenAIWrapper to create a client
-        oai_wrapper = OpenAIWrapper(**llm_config)
+        openai_client_cfg = None
+        if llm_config and llm_config.get("config_list") is not None and len(llm_config["config_list"]) > 0:
+            openai_client_cfg = llm_config["config_list"][0].copy()
+
+        oai_wrapper = OpenAIWrapper(base_config=openai_client_cfg)
```

Comment:
```suggestion
        oai_wrapper = OpenAIWrapper(**llm_config)
        llm_config = oai_wrapper._config_list[0]
```
We should not access private methods here, so my suggestion needs to be later modified when we add a public property.

Reply:
The modifications you suggested will lead to this error: NotFoundError: Error code: 404 - {'error': {'code': '404', 'message': 'Resource not found'}}